### DISCUSSION TITLE
Using sweetalert in bulk action confirmation

### DISCUSSION THREAD

**Discussion by sebbydrn** [rappasoft/laravel-livewire-tables#1711] [Q&A]
Is there a way that I can use sweetalert in the bulk action confirmation?

**Comment by lrljoe** [comment]
So bulk action confirmation is just the Livewire native wire:confirm

Not familiar with that package, but there's a few features upcoming that may possibly help, namely smoother wireable buttons, header action buttons.

What are you trying to achieve ultimately? I use the WireElements modal package for most of my modal needs, I'm more consistently available on Discord if you jump on there, and I'll try to give you some guidance
